### PR TITLE
Add instructions

### PR DESCRIPTION
**Pull request by the94air**
Temporarily install instructions for people who want to test it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,20 @@
 Your app to use emotes anywhere - simply point the app to a folder and click on one of the images.
 You can then paste it anywhere where images are supported.
 
+## How to install
+1. Clone and npm install
+```bash
+$ git clone https://github.com/lostdesign/teamotes.git && cd teamotes
+$ npm install
```
Comment (Andr3wRiv3rs) [inline README.md:12]:
```suggestion
git clone https://github.com/lostdesign/teamotes.git && cd teamotes
npm install
```

File: README.md [hunk 2]
```diff
@@ -5,6 +5,20 @@
...
+```
+## How to use
+1. Run the app
+```bash
+$ npm run dev
```
Comment (Andr3wRiv3rs) [inline README.md:17]:
```suggestion
npm run dev
```

**Comment by the94air** [comment]
Done @Andr3wRiv3rs 😁. Read to merge!

**Comment by the94air** [comment]
Thanks! :rocket:

**Pull request merged**